.github/DISCUSSION_TEMPLATE/new.yml

```diff
@@ -0,0 +1,61 @@
+name: New Prisma Discussion
```

Comment:
why not use "Question" to be more specific and clear about what exactly this is? 


```suggestion
name: Question
```